nck/readers/twitter/reader.py

```diff
@@ -311,23 +311,27 @@ def get_campaign_management_report(self):
             for entity_obj in ACCOUNT_CHILD_OBJECTS[self.entity]
         ]
 
+    @retry(stop=stop_after_delay(901))
```

Comment:
Why this magical 901 delay?

Reply:
Hi Vivien, I used 901 to avoid the rateLimit on the Twitter API ["900 requests per 15-minute interval can be made on behalf of each user"](https://developer.twitter.com/en/docs/twitter-api/v1/rate-limits). 15 * 60 = 900. I tested this solution locally for the same request and never faced a RateLimit issue anymore.

Reply:
Maybe we can add a comment to explain a little bit more.